apic.go

```diff
@@ -70,40 +69,40 @@ func yaml_string_read_handler(parser *yamlParser, buffer []byte) (n int, err err
 }
 
 // Reader read handler.
-func yaml_reader_read_handler(parser *yamlParser, buffer []byte) (n int, err error) {
+func yamlReaderReadHandler(parser *yamlParser, buffer []byte) (n int, err error) {
 	return parser.input_reader.Read(buffer)
 }
 
 // Set a string input.
-func yaml_parser_set_input_string(parser *yamlParser, input []byte) {
+func (parser *yamlParser) setInputString(input []byte) {
 	if parser.read_handler != nil {
 		panic("must set the input source only once")
 	}
-	parser.read_handler = yaml_string_read_handler
+	parser.read_handler = yamlStringReadHandler
 	parser.input = input
 	parser.input_pos = 0
 }
 
 // Set a file input.
-func yaml_parser_set_input_reader(parser *yamlParser, r io.Reader) {
+func (parser *yamlParser) setInputReader(r io.Reader) {
 	if parser.read_handler != nil {
 		panic("must set the input source only once")
 	}
-	parser.read_handler = yaml_reader_read_handler
+	parser.read_handler = yamlReaderReadHandler
 	parser.input_reader = r
```

Comment:
Any plan to rename them to


```suggestion
	parser.readHandler = yamlReaderReadHandler
 	parser.inputReader = r
```

There are many of these.

In this PR or another?

Reply:
Indeed, this can be another PR.